src/aics-image-viewer/shared/utils/controlPointsToLut.ts

```diff
@@ -71,3 +71,35 @@ export function initializeLut(
   aimg.setLut(channelIndex, lutObject);
   return newControlPoints;
 }
+
+export function controlPointsToRamp(controlPoints: ControlPoint[]): [number, number] {
+  if (controlPoints.length < 3) {
+    return [0, TFEDITOR_MAX_BIN];
+  }
+  return [controlPoints[1].x, controlPoints[2].x];
+}
```

Comment:
What does it mean if there are only two (or fewer) control points? Is one control point always at (0, 0.0), and the other at (255, 1.0)?

Reply:
Took this callout as an opportunity to make this function more robust (I knew I wasn't quite dotting all my i's the first time around) but yeah: all the lut generator functions in volume-viewer add those two points at the ends.